### ISSUE TITLE
Tabbed widget option

### ISSUE THREAD

**Issue by hockeymikey** [mpcjanssen/simpletask-android#1033]
The widget list should have an option to define one sort criteria as a tabbed option. So I could set by priority and there would be tabs on the top bar of the widget and tapping one filters the list by the tab (like separate pages). This would help declutter the widget and help users get the info they need quickly from the launcher. I could add mocks if it helps vizualize it.

**Comment by OpenWarp** [comment]
I would suggest a drop down arrow just left to the settings gear in the widget title. When creating a widget, you could choose what this arrow button will do,
1. Drop-down context list to filter on
2. Drop-down tag/project list to filter on
3. Sort by priority
4. Sort by due date

Although only options 1 and 3 are required in my opinion, the arrow button can be configured to sort or filter on whatever. Once set up, it will perform only one action on a single criteria.

**Comment by mpcjanssen** [comment]
Android widget programming sucks. And this is trying to do too much in the widget IMO. All these things are already easily doable from the app itself, shortcuts to specific filters on the homescreen make it even more direct.
So probably never going to happen.

**Comment by hockeymikey** [comment]
@mpcjanssen Challenge accepted.

**Comment by smichel17** [comment]
If you have simpletask-specific questions, you can ask here or in one of the chat rooms (linked [here](https://github.com/mpcjanssen/simpletask-android/blob/master/app/src/main/assets/index.en.md#support); they're all bridged together) and I'll do my best to help, depend on how much time I am around (I'm somewhat busy today).

**Comment by OpenWarp** [comment]
> Android widget programming sucks. And this is trying to do too much in the widget IMO. All these things are already easily doable from the app itself, shortcuts to specific filters on the homescreen make it even more direct.
> So probably never going to happen.

Fair enough. It is possible to live without widgets when shortcuts are available. I do not see any however when long press on the app icon. Quick access to three recent context filters through application actions (press and hold app icon) menu would suffice.

**Comment by mpcjanssen** [comment]
Create a saved filter and long press that. Then you can create a shortcut to the homescreen.
